pkg/utils/util.go

```diff
@@ -460,7 +466,21 @@ func RemovePrefixedParameters(param map[string]string) (map[string]string, error
 	newParam := map[string]string{}
 	for k, v := range param {
 		if strings.HasPrefix(k, csiParameterPrefix) {
-			// Check if its well known
+			// Check if it should be included
```

Comment:
this check seems to be redundant?

Reply:
Can you clarify how it is redundant? Are you referring to checking the prefix? From my understanding, we need this check to return an error if it's using the prefix but we don't know about it (if it doesn't match any of the cases in the second switch statement).

Reply:
Oh I see, good catch, yes this is redundant. Will remove.

Reply:
Undid the changes to this function, but updated the `util_test.go` to include the two other prefixed parameters that should be removed (weren't covered in tests).